extensions/wikia/MonetizationModule/MonetizationModuleHelper.class.php

```diff
@@ -203,6 +234,26 @@ public function setThemeSettings( $adUnits, $memcKey ) {
 	}
 
 	/**
+	 * Set memcache data (json format)
+	 * @param string $memcKey
+	 * @param array $data
+	 * @param array $loggingParams
+	 * @param boolean $setMemc - set to true to set data to memcache
+	 */
+	public function setMemcache( $memcKey, $data, $loggingParams, $setMemc = true ) {
```

Comment:
the `$setMemc` var seems odd in this method. Why would I pass in a `false` when calling this method? Instead of checking the `$setMemc` var and not call the method in the first place?